build.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env sh
+echo '[INFO] Feral Installer - (c) Copyright 2020 Maximilian Götz (https://www.maxbits.net/)'
+echo '[INFO] Please make sure that Git, CMake, LibGMP and LibMPFR are installed!'
+echo '[INFO] Installation started. Checking for Git, sudo and CMake...'
+
+# Check for Git
+if hash git 2>/dev/null; then
+    echo '[INFO] Git found.'
+else
+    echo '[FATAL] Git not available! Install it to continue! Aborting...'
```

Comment:
I'm not an expert but shouldn't it be **is** here and down below?

Reply:
You're right, but I left them out intentionally to make the messages a little shorter. If you think that the *is* is important, feel free to add it! :)

Reply:
Nah, I don't. Just noticed, that's cool! 👍